pkgs/tools/misc/diffoscope/default.nix

```diff
@@ -54,8 +55,9 @@ python3Packages.buildPythonApplication rec {
       python-magic progressbar33 pypdf2 tlsh
     ])
     ++ lib.optionals stdenv.isLinux [ python3Packages.pyxattr python3Packages.rpm acl cdrkit dtc ]
+    ++ lib.optionals (enableBloat && enableUnfree) [ apktool ]
```

Comment:
```suggestion
```

Reply:
I think it makes more sense to make it require both options.  It's not something most users are likely to need, so it's bloat, and it's unfree.  Just because we don't currently have any non-bloat unfree doesn't mean that that category doesn't make sense.

Reply:
I just moved it within the enableBloat conditional path

Reply:
Done